### ISSUE TITLE
Negative integers don't work

### ISSUE THREAD

**Issue by unindented** [pateketrueke/json-schema-faker#45]
This doesn't generate a negative integer:

```json
{
  "type": "integer",
  "minimum": -100,
  "maximum": 0
}
```

Am I doing something wrong?

**Comment by pateketrueke** [comment]
Confirmed, It's a bug.

I'll work on this asap, thanks for your feedback.

**Comment by unindented** [comment]
Thank you!

**Comment by pateketrueke** [comment]
I've investigated a bit and this works perfectly:

```json
{
  "type": "integer",
  "minimum": -100,
  "maximum": 1
}
```

Let me check out what the json-schema spec says. :beers:

**Comment by unindented** [comment]
I knew it wasn't you. :smile:

**Comment by philberg** [comment]
I have fixed this issue in [my PR](https://github.com/Marak/faker.js/pull/218) and added an explicit test

**Comment by pateketrueke** [comment]
Fixed on `0.1.12`, thanks to @philberg yay! :beers:
